src/viur_cli/install.py

```diff
@@ -55,15 +55,68 @@ def get_version_info(software: str, version: str) -> tuple[str, str]:
 
 @cli.group()
 def install():
-    """Install ViUR features"""
+    """
+    Manage the installation of ViUR features.
+
+    The 'install' command group provides various subcommands for managing the installation of ViUR features, including
+    the ViUR administration interfaces and the scriptor IDE. You can choose to install specific versions or use the
+    latest available versions.
+
+    Example Usage:
+    ```shell
+    viur install vi --version v2.0.0
+    viur install admin
+    viur install scriptor --version latest
+    ```
+
+    Available subcommands:
+    - 'vi': Install the legacy ViUR administration interface.
+    - 'admin': Install the new ViUR administration interface.
+    - 'scriptor': Install the scriptor IDE.
+
+    Note:
+    - When using the 'admin' subcommand, the new administration interface will be installed.
+    - The 'scriptor' subcommand installs the scriptor IDE.
+    - Ensure that you have a valid project configuration before using these subcommands.
+
+    :return: None
+    """
 
 
 @install.command()
 @click.argument("version", default="latest")
 @click.option("--next", "-n", "next_", is_flag=True, default=False)
 @click.option("--target", "-t", default="vi")
 def vi(version, target, next_):
-    """Install vi — the legacy ViUR administration interface."""
+    """
+        Install the legacy ViUR administration interface.
+
+        This subcommand allows you to install the legacy ViUR administration interface. You can specify a version to
+        install, or use the latest available version. It includes a 'next' option for compatibility,
+        but it is deprecated in favor of the 'admin' subcommand.
+
+        :param version: str, default: 'latest'
+            The version of the legacy ViUR administration interface to install.
+            Use 'latest' for the latest available version.
+
+        :param target: str, default: 'vi'
+            The target folder where the ViUR administration interface will be installed.
+
+        :param next_: bool, default: False
+            A deprecated option, please use the 'admin' subcommand for the new administration interface.
+
+        Example Usage:
+        ```shell
+        viur install vi --version v2.0.0 --target my-admin
+        viur install vi --version latest
+        ```
+
+        The 'vi' subcommand downloads and installs the specified
+        or latest version of the legacy ViUR administration interface.
+        It updates the configuration with the installed version.
+
+        :return: None
+        """
```

Comment:
```suggestion
    """Install the legacy ViUR administration interface.

    This subcommand allows you to install the legacy ViUR administration interface. You can specify a version to
    install, or use the latest available version. It includes a 'next' option for compatibility,
    but it is deprecated in favor of the 'admin' subcommand.

    :param version: str, default: 'latest'
        The version of the legacy ViUR administration interface to install.
        Use 'latest' for the latest available version.

    :param target: str, default: 'vi'
        The target folder where the ViUR administration interface will be installed.

    :param next_: bool, default: False
        A deprecated option, please use the 'admin' subcommand for the new administration interface.

    Example Usage:
    ```shell
    viur install vi --version v2.0.0 --target my-admin
    viur install vi --version latest
    ```

    The 'vi' subcommand downloads and installs the specified
    or latest version of the legacy ViUR administration interface.
    It updates the configuration with the installed version.

    :return: None
    """
```